src/components/Footer/index.js

```diff
@@ -34,23 +57,22 @@ const Footer = () => (
     <Disclaimer>
       <Row>
         <Column span={12}>
-          <Heading as="h2">Lukt het niet om een melding te doen?</Heading>
-          <Paragraph>&nbsp;</Paragraph>
+          <StyledHeading as="h4">Lukt het niet om een melding te doen?</StyledHeading>
         </Column>
 
         <Column span={12}>
-          <Paragraph>
+          <StyledText>
             Bel het Gemeentelijk informatienummer: 14 020 <br />
             op werkdagen van 08.00 tot 18.00 uur.
-          </Paragraph>
+          </StyledText>
         </Column>
       </Row>
     </Disclaimer>
 
     <Privacy>
       <Row>
         <Column span={12}>
-          <Link href="https://www.amsterdam.nl/privacy/">Privacy</Link>
+          <StyledLink size={5} href="https://www.amsterdam.nl/privacy/" variant="with-chevron">Privacy</StyledLink>
```

Comment:
What does the `size` prop do? I don't see that prop being available in the [`Link` component](https://github.com/Amsterdam/amsterdam-styled-components/blob/master/packages/asc-ui/src/components/Link/Link.tsx).

Reply:
was typo: gedaan